xyzspaces/spaces.py

```diff
@@ -1326,3 +1326,37 @@ def add_features_geopandas(
                 features_size=features_size,
                 chunk_size=chunk_size,
             )
+
+    def clone(self, space_id: str = None):
+        """
+        To copy current space data into a new space.
```

Comment:
remove `To` just write `copy current space data into a new space`.

Reply:
Done changes